frontend/src/sagas.js

```diff
@@ -587,58 +595,59 @@ export function* executeConfirmedActionsSaga({ showLoading, projectId, subprojec
     };
     for (const index in actions) {
       const action = actions[index];
-      const resource = action.intent.split(".", 1)[0];
       try {
-        switch (resource) {
-          case "project":
-            switch (action.action) {
-              case "grant":
-                yield callApi(api.grantProjectPermissions, action.id, action.intent, action.identity);
-                permissionsChange.projectId = action.id;
-                break;
-              default:
-                break;
-            }
+        switch (action.intent) {
```

Comment:
I renamed it to executeActions because it should be able to execute all actions if its in the right format, no matter if actions are additional ones or not.